erpnext/accounts/doctype/pos_session_invoices/pos_session_invoices.json

```diff
@@ -0,0 +1,108 @@
+{
```

Comment:
Why child table in POS Session? Since an Invoice can have only one POS Session, it makes sense to link the POS Session to the invoice and not add a child table. Maybe a button in POS Session can be added to view invoices linked to this session.

Also doctype names are always in singular, like "POS Session Invoice" not invoices. Since now row represents only one invoice :)